AugurProject/augur
# Current dispute window component

https://github.com/AugurProject/augur/issues/2922

to test drop something like this:
```
<WindowProgress
          title="Current Dispute Window"
          description="Few lines explaining the purpose of the Dispute Window"
          startTime={Date.parse("September 1, 2019 05:33:00") / 1000}
          endTime={Date.parse("September 8, 2019 05:33:00") / 1000}
          currentTime={Date.now() / 1000}
          countdownLabel="time remaining in window"
        />
```
into something like market-view to see an example.
